src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -253,13 +261,52 @@ const ShareWorkspaceModal = ({ onDismiss, workspace, workspace: { workspace: { n
     searchValueValid && !searchHasFocus && p(addUserReminder),
     h2({ style: { ...Style.elements.sectionHeader, margin: '1rem 0 0.5rem 0' } }, ['Current Collaborators']),
     div({ ref: list, role: 'list', style: styles.currentCollaboratorsArea }, [
-      h(Fragment, _.map(renderCollaborator, Utils.toIndexPairs(acl))),
+      h(Fragment, _.flow(
+        _.remove(aclEntryIsTerraSupport),
+        Utils.toIndexPairs,
+        _.map(renderCollaborator)
+      )(acl)),
       !loaded && centeredSpinner()
     ]),
     updateError && div({ style: { marginTop: '1rem' } }, [
       div(['An error occurred:']),
       updateError
     ]),
+    div({ style: { ...modalStyles.buttonRow, justifyContent: 'space-between' } }, [
+      h(IdContainer, [
+        id => h(TooltipTrigger, {
+          content: terraSupportAccess ?
+            `Terra Support has ${_.toLower(terraSupportAccess.accessLevel)} access to this workspace` :
+            'Grant Terra Support reader access to this workspace'
+        }, [
+          label({ htmlFor: id }, [
+            h(Switch, {
+              id,
+              checked: !!terraSupportAccess,
+              onChange: checked => {
+                if (checked) {
+                  addTerraSupportToAcl()
+                } else {
+                  removeTerraSupportFromAcl()
+                }
+              }
+            }),
+            span({ style: { marginLeft: '1ch' } }, 'Share with support')
```

Comment:
Did UX give the design with the label after the switch? Personally, I think it would be more standard to have the label before the control.

Reply:
Yes, the label is after the switch in the UX design attached to [SU-8](https://broadworkbench.atlassian.net/browse/SU-8).

I could go either way here. I agree that "Share with support? True/False" reads more naturally. On the other hand, checkboxes usually appear before their label and the switch is a fancy checkbox. We don't currently have an established pattern for this in Terra since the Switch component is only used in one other place (entity editor in data tables) and there it doesn't have a label other than the True/False in the switch itself.

Reply:
Order changed in 596956da4fd55460df7c7de70c786934bba40962.

Reply:
![Screen Shot 2022-03-24 at 10 44 22 AM](https://user-images.githubusercontent.com/1156625/159942128-66d1ef1f-404e-4243-afe5-b6063551cf29.png)